### DISCUSSION TITLE
Annotations in Structure/Symbol Names?

### DISCUSSION THREAD

**Discussion by jmcarter9t** [NationalSecurityAgency/ghidra#4311] [Ideas]
Scenario: Reversing C++ code with many vtables. In order to get meaningful descriptions for computed calls to the functions in those vtables I created a structure for a specific vtable with function pointers for the DataType and create a descriptive name for each function whose description allows me to understand which function gets called (currently I include the function entry point address). 

Current xref solution: I can also insert a comment with an annotation to the address of the function just above the call in the decompilation -- this allows me to jump quickly to the xref. 

Idea: What might be nice is to have an ANNOTATION IN THE NAME I use for the function in the structure, so the comment becomes unnecessary and it would automatically be used each time I set the type for that vtable.

Is there an alternative approach I do not know (highly possible)?  Is this possible?  
Thanks!

**Comment by dragonmacher** [comment]
More generically, it may be useful to allow for the editing of any field to assign a hyperlink style of navigation.  Such a feature would work for this case.
